GenerateCryptoPIN length output inconsistent

I received this failed test for the `GenerateCryptoPIN` method on the first run but subsequent runs were successful. I am wondering if there is something about the algorithm that does or cannot guarantee the correct length every time. Another issue, perhaps separate and perhaps related, is whether the PIN can begin with zero (this *should* be supported and is supported with the psuedo-random pin generator).

```
--- FAIL: TestGenerateCryptoPIN (0.00s)
    --- FAIL: TestGenerateCryptoPIN/4_character_random_pin (0.00s)
        cryptohelper_test.go:74: expected len(4) but got (3)
FAIL
exit status 1
FAIL	github.com/gregtczap/cryptohelper	0.001s
```

--- a/cryptohelper.go
+++ b/cryptohelper.go
@@ -58,7 +58,9 @@ func GenerateCryptoPIN(length int) (string, error) {
 		return "", err
 	}
 
-	return fmt.Sprint(pin), nil
+	fmtStr := "%0" + strconv.Itoa(length) + "v"
+
+	return fmt.Sprintf(fmtStr, pin), nil
 }
 
 // GenerateCryptoSequence returns a cryptographically secure psuedo-random sequence of bytes of the

